README.md

```diff
@@ -1,18 +1,8 @@
-# svelte app
+# svelte playground
```

Comment:
I think this needs proper title formatting. Not sure if you were going for something else on purpose though.